### PR TITLE
PROGMI-215 Add more BrowserSync options

### PR DESCRIPTION
**Pull request by thiagodemellobueno**
This is for https://github.com/kalamuna/progenity.com/pull/187

### CODE REVIEW

**Comment by RobLoach** [comment]
@thiagodemellobueno Nice!  We can now change `bsWebroot` in `kalastatic.yml`... This will make it so we can host from `web` in PROGMI, rather then `web/kalastatic`. Mind another QA and then merge?

**Comment by soniktrooth** [comment]
@RobLoach does this mean that all KS files will be scattered inside the Drupal root or will they all still be in a folder?

**Comment by RobLoach** [comment]
It gives you the option to change how BrowserSync operates. It doesn't split kalastatic files up, just allows you to change how npm start works.

File: bin/kalastatic [hunk 1]
```diff
@@ -66,9 +68,10 @@ program
 
       // Set up the server.
       bs.init({
-        server: finalDest,
-        index: 'index.html',
-        serveStatic: ['.']
+        server: base,
```
Comment (RobLoach) [inline bin/kalastatic]: It is handy, the more configs the more easier we can adapt to change. I like the options you added :+1:

**Pull request merged**